### DISCUSSION TITLE
forum question

### DISCUSSION THREAD

**Discussion by liot1** [community/community#21377] [Other Feature Feedback, Questions, & Ideas]
<p>hello,</p>
<p>i’m fairly new to the vpn, security browsers, etc.</p>
<p>I’ve downloaded mullvad vpn, the first day i was able to test with am.i.mullvad.  but since then, I am not receiving any replies, it’s just sitting in the query state asking to be patient while it checks, but nothing ever results.  I don’t know if this is where to ask.  I’ve ck’d the mullvad site and cannot find any way to correspond. </p>
<p>I searched Github and found threads for mullvad, but do not know where i can submit this issue.</p>
<p>thank you!</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>Did you try emailing Mullvad?  <a href="mailto:support@mullvad.net">support@mullvad.net</a></p>
